src/Components/AddRoom/AddRoom.tsx

```diff
@@ -0,0 +1,221 @@
+
+import { Box, Button, Container, FormControl, Grid, Paper, TextField, Typography } from '@mui/material'
+import * as React from 'react'
+import { ThemeProvider, createTheme, styled } from '@mui/material/styles';
+import axios from 'axios';
+import { useForm } from 'react-hook-form';
+import Select from 'react-dropdown-select';
+import FileUploadOutlinedIcon from '@mui/icons-material/FileUploadOutlined';
+import { useNavigate } from 'react-router';
+
+
+export default function AddRoom() {
+  const { register, handleSubmit, formState: { errors } } = useForm()
+  let reqHeaders = 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2NThhMTgyYjQ3ZWUyYjE0Zjk1NDY5OTAiLCJyb2xlIjoiYWRtaW4iLCJ2ZXJpZmllZCI6ZmFsc2UsImlhdCI6MTcwNDUzNjEyMCwiZXhwIjoxNzA1NzQ1NzIwfQ.p15lXfscJSFl8OJ4drIUj0vPPS3nO4L_U6iTbtwBdf8'
+  let Headers = { Authorization: reqHeaders }
+  const [facilities, setFacilities] = React.useState([{ "value": '', 'label': '' }])
+  const [selectedValue, setSelectedValue] = React.useState([])
+  const [imgs, setImgs] = React.useState(''),
+    handleImage = (e) => {
+      setImgs(e.target.files[0])
+    }
+    const navigate=useNavigate()
+  const theme = createTheme({
+    components: {
+      MuiFilledInput: {
+        styleOverrides: {
+          underline: {
+            '&:after': {
+              borderBottom: 'none',
+            },
+            '&:before': {
+              borderBottom: 'none',
+            },
+          },
+        },
+      },
+    },
+
+  });
+  const VisuallyHiddenInput = styled('input')({
+    clip: 'rect(0 0 0 0)',
+    clipPath: 'inset(50%)',
+    height: 1,
+    overflow: 'hidden',
+    position: 'absolute',
+    bottom: 0,
+    left: 0,
+    whiteSpace: 'nowrap',
+    width: 1,
+  });
+
+  {/*Add Room Function */ }
+
+  const AddNewRoom = (data) => {
+    const formattedSelected = selectedValue.map(({ value }) => value)
+    axios.post('http://154.41.228.234:3000/api/v0/admin/rooms', { ...data, imgs: data.imgs[0], facilities: formattedSelected }, { headers: { ...Headers, "Content-Type": "multipart/form-data" } }).then((response) => {
+      console.log(response);
```

Comment:
Don'tDont't forget to remove logs and add toast messages in case of success or failure for better user experience and feedback.